tests/packages/cmake_defines/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 3.26)
```

Comment:
```suggestion
cmake_minimum_required(VERSION 3.15)
```
Should add a badge for the minimum CMake

Reply:
Done, thanks!